test/forms/mappings/ConstraintsSpec.scala

```diff
@@ -156,4 +156,44 @@ class ConstraintsSpec extends WordSpec with MustMatchers with Constraints with R
 
     behave like regexWithValidAndInvalidExamples(companyUniqueTaxReference, validUtr, invalidUtr, invalidMsg, utr)
   }
+
+  "email" must {
+
+    val validEmail = Table(
+      "a@email.com",
+      "a@bc",
+      "123@456"
+    )
+
```

Comment:
First row in Table should be a title not a test value. So should be something like "email". Same for invalidEmail - change "value" to "email".